src/main/java/calculator/StringAddCalculator.java

```diff
@@ -0,0 +1,58 @@
+package calculator;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+    public static final String DEFAULT_DELIMITER = "[,:]";
+    public static final Pattern CUSTOM_DELIMITER_PATTERN = Pattern.compile("//(.)\\n(.*)");
+    private String input;
+    private String delimiter = DEFAULT_DELIMITER;
```

Comment:
```suggestion
    public static final String DEFAULT_DELIMITER = "[,:]";
    public static final Pattern CUSTOM_DELIMITER_PATTERN = Pattern.compile("//(.)\\n(.*)");

    private String input;
    private String delimiter = DEFAULT_DELIMITER;
```

상수와 인스턴스 변수 사이에는 한 줄을 띄워 주세요~